packages/story-editor/src/app/pageCanvas/utils/storyPageToNode.tsx

```diff
@@ -50,83 +50,101 @@ const Page = styled.div`
   outline: 0;
 `;
 
-const PreviewWrapper = styled.div`
+// @todo This casting below is incorrect. Not sure what's the correct way here since it works as-is but TS claims that it shouldn't.
+const PreviewWrapper = styled.div<{ background: Pattern }>`
   height: 100%;
   position: relative;
   overflow: hidden;
   background-color: ${({ theme }) => theme.colors.fg.white};
   border-radius: 4px;
-  ${({ background }) => generatePatternStyles(background)}
+  ${({ background }) => generatePatternStyles(background) as string}
```

Comment:
@barklund Thoughts about this one here? I get `Interpolation` not matching `CSSProperties` error here, however, it works actually. Thoughts?

Reply:
If we just change the return type of `generatePatternStyles` to `Record<string, string>`, everything seems to compile fine. It's probably also more correct, as I don't think styled would appreciate non-string values.